services/src/main/java/org/keycloak/theme/DefaultThemeManager.java

```diff
@@ -72,7 +73,7 @@ public Theme getTheme(String name, Theme.Type type) {
         if (theme == null) {
             theme = loadTheme(name, type);
             if (theme == null) {
-                theme = loadTheme("keycloak", type);
+                theme = loadTheme("keycloak.v2", type);
```

Comment:
I'm not sure if it isn't confusing that all themes (even the old ones) use `v2` with the old `keycloak` still available. The problem I see is that `keycloak` and `keycloak.v2` are technically same themes (except account console) but the `v2` could make the impression something's new in all themes.

But I'm ok with this if @stianst is.

Reply:
This line of code is only used when you haven't selected any theme.  It shows up in the UI as "Select one".

You do see both keycloak and keycloak.v2 in the dropdown when you go to select a theme.  But that actually turns out kind of nice because there is some symmetry.   And without it, the code above wouldn't work.  You would need special logic to know that if it's a certain type then you have to fall back to "keycloak".

Reply:
Yes, the line I selected here is not very accurate. My comment above is more generic – if it's ok to have identical `keycloak` and `keycloak.v2`. I'll wait for @stianst decision here.